backend/src/entities/tags.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Tag {
```

Comment:
What does this object represent and when is it used in the prototype?

Reply:
Where would the links point to? In the prototype it seems to be static text with no onClick functionality.

Reply:
We weren't sure if there would be links on these or not. We figured they kind of looked like they could contain links so there's no harm in having the field. But we can remove it need be. There's not much harm in having an extra field though